src/components/DateInput.jsx

```diff
@@ -220,7 +224,11 @@ class DateInput extends React.PureComponent {
             focused && styles.DateInput_input__focused,
             disabled && styles.DateInput_input__disabled,
           )}
-          aria-label={ariaLabel === undefined ? placeholder : ariaLabel}
+          aria-label={
+            ariaLabel === undefined 
+              ? ariaLabelledBy ? undefined : placeholder 
```

Comment:
Could we maybe do this in an intermediate variable? Nested ternaries are very hard to follow.

Reply:
I separated one ternary into a variable and left the original simplified variable in place. What do you think?